chart/etcd-backup-restore/values.yaml

```diff
@@ -153,3 +153,6 @@ backupRestoreTLS: {}
 
 # podAnnotations that will be passed to the resulting etcd pod
 podAnnotations: {}
+
+# sseCustomerKey is the AES Key used to enable Server Side Encryption
```

Comment:
Since `sseCustomerKey` is an optional field, could you comment it out and add a corresponding comment which notifies that this is an optional field as seen here -

https://github.com/gardener/etcd-backup-restore/blob/53356a0b82a4957110725e43d4f3f0f41b3aaa46/chart/etcd-backup-restore/values.yaml#L91-L98

Notify that the field expects a string as shown elsewhere in the file.

Reply:
Agree. I've commented it out now and called out that it needs to be a string.